src/json.hpp

```diff
@@ -6101,7 +6101,9 @@ class serializer
     @param[in] ichar  indentation character to use
     */
     serializer(output_adapter_t<char> s, const char ichar)
-        : o(std::move(s)), loc(std::localeconv()),
+        : o(std::move(s)),
+          number_buffer(),
```

Comment:
What is the warning that led to adding `number_buffer()` here? The member is initialized in the definition:

```cpp
std::array<char, 64> number_buffer{{}};
```

Reply:
It was bad icc check -- I think this change is not really necessary.

Reply:
Thanks for checking back. Could you revert this change please?